src/Queries/Where/Where.ts

```diff
@@ -59,7 +59,7 @@ type WhereTypes = {
   };
 };
 
-const operators = [
+const operators: string[] = [
```

Comment:
I just noticed that this removes the type safety. What do you think about reverting, and just cast `operator` as `as (typeof operators)[number]` when it's defined in line `237`?

Reply:
Yes, we can. By the way, what do you think if we go straight to getting it from the isOperator object directly, for example
```typescript
const operator = description as (typeof operators)[number];
if (operator && isOperator[operator]?.call(value)) {
  // ...
}
```

This would be clearer and more efficient.

Reply:
Nope, there is no advantage between any,
I already uploaded the changes